lib/elixir/lib/dynamic_supervisor.ex

```diff
@@ -0,0 +1,684 @@
+defmodule DynamicSupervisor do
+  @moduledoc ~S"""
+  A supervisor that starts children dynamically.
+
+  The `Supervisor` was designed to handle mostly static children, that
```

Comment:
> The `Supervisor` was designed to handle mostly static children...

I found this confusing: your use of "the" makes it sound like you are referring to a specific supervisor, but it's not clear *which* specific supervisor you are referring to.  After re-reading this I realized that you are talking about the `Supevisor` module in general.  WDYT about changing this to:

> The `Supervisor` API was designed to handle mostly static children...

Reply:
Can we use "module" instead of "API"?

Reply:
Sure, that works!